wideq/core.py

```diff
@@ -12,7 +12,7 @@
 from requests.adapters import HTTPAdapter
 from requests.packages.urllib3.util.retry import Retry
 
-GATEWAY_URL = 'https://kic.lgthinq.com:46030/api/common/gatewayUriList'
+GATEWAY_URL = 'https://kic.lgthinq.com:46030'
```

Comment:
Let's leave this change where we split up the base URL and path out for now—it looks interesting, but I would love to review it carefully in a separate PR.